stories/settings.stories.js

```diff
@@ -70,7 +76,7 @@ const setupSettings = () => {
 				created: '2016-12-06T14:48:54.188Z',
 				dataRetentionResetOnNewActivity: true,
 				dataRetentionTtl: 'MONTHS_26',
-				defaultProfileId: '12345678',
+				defaultProfileID: '12345678',
```

Comment:
I think this needs to remain as is, since it is used to mock a 3P API response.

Reply:
Whoops, okay. Changing it has no effects on the tests, but if it's a mock response that makes sense 👍

Reply:
@felixarntz there are other keys like this already in develop that have been changed to use our casing that shouldn't have been. I'll open a separate PR for this.